invenio_app_rdm/theme/templates/semantic-ui/invenio_app_rdm/frontpage.html

```diff
@@ -36,11 +37,10 @@ <h3 class="section-title">{{ _("Configure") }}</h3>
           </div>
           <h3 class="section-title">{{ _("Customize") }}</h3>
             <div class="command-line">
-              {% trans configuration_link="https://inveniordm.docs.cern.ch/customize/styling/", extensions_link="https://inveniordm.docs.cern.ch/extensions/", custom_extension_link="https://inveniordm.docs.cern.ch/extensions/custom/" %}Customize
-              the layout and adapt the styling to match your institution's. See how to do so <a
-                href="{{ configuration_link }}">here</a>. You can also add new functionality using <a
-                href="{{ extensions_link }}">supported plugins</a>, or <a href="{{ custom_extension_link }}">writing
-                your own</a>.{% endtrans %}
+              {% trans configuration_link="https://inveniordm.docs.cern.ch/customize/styling/" %}
```

Comment:
If we want to remove the `customize` link from here, I would put it above. Then change this step name to `Style` or `Theming` or similar. Therefore:

1- Customize: permissions, data model, etc.
2- Style it
3- Deploy it